k8s/namespaces/wa/kustomization.yaml

```diff
@@ -7,4 +7,5 @@ bases:
 - wa-task-management-api/wa-task-management-api.yaml
 - wa-task-configuration-api/wa-task-configuration-api.yaml
 - wa-proto-frontend/wa-proto-frontend.yaml
+- wa-standalone-task-bpmn/wa-standalone-task-bpmn.yaml
```

Comment:
is this supposed to be in all environments? do you have the needed approvals?

Reply:
its a new repo just used to hold tests

Reply:
then it shouldn't be going in to prod, see the line below

Reply:
Hi @timja , yes, we need this service in Prod too. Thanks.